default.webpack.config.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
What's the reason for adding the babel-loader info here and the babel-loader stuff below? It looks like it duplicates the info?